### ISSUE TITLE
mame 0.249: compilation error

### ISSUE THREAD

**Issue by 0-wiz-0** [mamedev/mame#10482]
I see a new compilation problem in 0.249 (not there in 0.248):
```
Compiling src/devices/bus/ieee488/hp9122c.cpp...
In file included from ../../../../../src/devices/bus/epson_qx/keyboard/keyboard.cpp:9:
../../../../../src/devices/cpu/mcs48/mcs48.h:74:20: error: expected constructor, destructor, or type conversion before '(' token
   74 | DECLARE_DEVICE_TYPE(I8021,   i8021_device)    // 1k internal ROM,       64 bytes internal RAM
      |                    ^
In file included from ../../../../../src/devices/bus/epson_qx/keyboard/keyboard.cpp:9:
../../../../../src/devices/cpu/mcs48/mcs48.h:512:1: error: expected class-name before '{' token
  512 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:519:21: error: 'machine_config' does not name a type
  519 |  i8021_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:519:63: error: 'device_t' has not been declared
  519 |  i8021_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:519:80: error: 'uint32_t' has not been declared
  519 |  i8021_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:523:10: error: 'uint64_t' does not name a type
  523 |  virtual uint64_t execute_clocks_to_cycles(uint64_t clocks) const noexcept override { return (clocks + 30 - 1) / 30; }
      |          ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:524:10: error: 'uint64_t' does not name a type
  524 |  virtual uint64_t execute_cycles_to_clocks(uint64_t cycles) const noexcept override { return (cycles * 30); }
      |          ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h: In member function 'auto i8021_device::p0_in_cb()':
../../../../../src/devices/cpu/mcs48/mcs48.h:515:27: error: 'bus_in_cb' was not declared in this scope; did you mean 'p0_in_cb'?
  515 |  auto p0_in_cb() { return bus_in_cb(); }
      |                           ^~~~~~~~~
      |                           p0_in_cb
../../../../../src/devices/cpu/mcs48/mcs48.h: In member function 'auto i8021_device::p0_out_cb()':
../../../../../src/devices/cpu/mcs48/mcs48.h:516:28: error: 'bus_out_cb' was not declared in this scope; did you mean 'p0_out_cb'?
  516 |  auto p0_out_cb() { return bus_out_cb(); }
      |                            ^~~~~~~~~~
      |                            p0_out_cb
../../../../../src/devices/cpu/mcs48/mcs48.h: At global scope:
../../../../../src/devices/cpu/mcs48/mcs48.h:528:1: error: expected class-name before '{' token
  528 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:531:21: error: 'machine_config' does not name a type
  531 |  i8022_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:531:63: error: 'device_t' has not been declared
  531 |  i8022_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:531:80: error: 'uint32_t' has not been declared
  531 |  i8022_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:535:10: error: 'uint64_t' does not name a type
  535 |  virtual uint64_t execute_clocks_to_cycles(uint64_t clocks) const noexcept override { return (clocks + 30 - 1) / 30; }
      |          ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:536:10: error: 'uint64_t' does not name a type
  536 |  virtual uint64_t execute_cycles_to_clocks(uint64_t cycles) const noexcept override { return (cycles * 30); }
      |          ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:540:1: error: expected class-name before '{' token
  540 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:543:21: error: 'machine_config' does not name a type
  543 |  i8035_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:543:63: error: 'device_t' has not been declared
  543 |  i8035_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:543:80: error: 'uint32_t' has not been declared
  543 |  i8035_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:547:1: error: expected class-name before '{' token
  547 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:550:21: error: 'machine_config' does not name a type
  550 |  i8048_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:550:63: error: 'device_t' has not been declared
  550 |  i8048_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:550:80: error: 'uint32_t' has not been declared
  550 |  i8048_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:554:1: error: expected class-name before '{' token
  554 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:557:21: error: 'machine_config' does not name a type
  557 |  i8648_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:557:63: error: 'device_t' has not been declared
  557 |  i8648_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:557:80: error: 'uint32_t' has not been declared
  557 |  i8648_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:561:1: error: expected class-name before '{' token
  561 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:564:21: error: 'machine_config' does not name a type
  564 |  i8748_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:564:63: error: 'device_t' has not been declared
  564 |  i8748_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:564:80: error: 'uint32_t' has not been declared
  564 |  i8748_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:568:1: error: expected class-name before '{' token
  568 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:571:21: error: 'machine_config' does not name a type
  571 |  i8039_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:571:63: error: 'device_t' has not been declared
  571 |  i8039_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:571:80: error: 'uint32_t' has not been declared
  571 |  i8039_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:575:1: error: expected class-name before '{' token
  575 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:578:21: error: 'machine_config' does not name a type
  578 |  i8049_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:578:63: error: 'device_t' has not been declared
  578 |  i8049_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:578:80: error: 'uint32_t' has not been declared
  578 |  i8049_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:582:1: error: expected class-name before '{' token
  582 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:585:21: error: 'machine_config' does not name a type
  585 |  i8749_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:585:63: error: 'device_t' has not been declared
  585 |  i8749_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:585:80: error: 'uint32_t' has not been declared
  585 |  i8749_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:589:1: error: expected class-name before '{' token
  589 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:592:21: error: 'machine_config' does not name a type
  592 |  i8040_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:592:63: error: 'device_t' has not been declared
  592 |  i8040_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:592:80: error: 'uint32_t' has not been declared
  592 |  i8040_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:596:1: error: expected class-name before '{' token
  596 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:599:21: error: 'machine_config' does not name a type
  599 |  i8050_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:599:63: error: 'device_t' has not been declared
  599 |  i8050_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:599:80: error: 'uint32_t' has not been declared
  599 |  i8050_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:603:1: error: expected class-name before '{' token
  603 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:606:22: error: 'machine_config' does not name a type
  606 |  mb8884_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                      ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:606:64: error: 'device_t' has not been declared
  606 |  mb8884_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:606:81: error: 'uint32_t' has not been declared
  606 |  mb8884_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:610:1: error: expected class-name before '{' token
  610 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:613:21: error: 'machine_config' does not name a type
  613 |  n7751_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:613:63: error: 'device_t' has not been declared
  613 |  n7751_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:613:80: error: 'uint32_t' has not been declared
  613 |  n7751_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:617:1: error: expected class-name before '{' token
  617 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:620:22: error: 'machine_config' does not name a type
  620 |  m58715_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                      ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:620:64: error: 'device_t' has not been declared
  620 |  m58715_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:620:81: error: 'uint32_t' has not been declared
  620 |  m58715_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:625:1: error: expected class-name before '{' token
  625 | {
      | ^
../../../../../src/devices/cpu/mcs48/mcs48.h:628:2: error: 'uint8_t' does not name a type; did you mean 'uid_t'?
  628 |  uint8_t upi41_master_r(offs_t offset);
      |  ^~~~~~~
      |  uid_t
../../../../../src/devices/cpu/mcs48/mcs48.h:629:22: error: 'offs_t' has not been declared
  629 |  void upi41_master_w(offs_t offset, uint8_t data);
      |                      ^~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:629:37: error: 'uint8_t' has not been declared
  629 |  void upi41_master_w(offs_t offset, uint8_t data);
      |                                     ^~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:633:25: error: 'machine_config' does not name a type
  633 |  upi41_cpu_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock, int rom_size, int ram_size);
      |                         ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:633:50: error: 'device_type' has not been declared
  633 |  upi41_cpu_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock, int rom_size, int ram_size);
      |                                                  ^~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:633:85: error: 'device_t' has not been declared
  633 |  upi41_cpu_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock, int rom_size, int ram_size);
      |                                                                                     ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:633:102: error: 'uint32_t' has not been declared
  633 |  upi41_cpu_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock, int rom_size, int ram_size);
      |                                                                                                      ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:635:25: error: 'master_callback' has not been declared
  635 |  TIMER_CALLBACK_MEMBER( master_callback );
      |                         ^~~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:635:2: error: ISO C++ forbids declaration of 'TIMER_CALLBACK_MEMBER' with no type [-fpermissive]
  635 |  TIMER_CALLBACK_MEMBER( master_callback );
      |  ^~~~~~~~~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:642:22: error: 'machine_config' does not name a type
  642 |  i8041a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                      ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:642:64: error: 'device_t' has not been declared
  642 |  i8041a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:642:81: error: 'uint32_t' has not been declared
  642 |  i8041a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:649:22: error: 'machine_config' does not name a type
  649 |  i8741a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                      ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:649:64: error: 'device_t' has not been declared
  649 |  i8741a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:649:81: error: 'uint32_t' has not been declared
  649 |  i8741a_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:656:23: error: 'machine_config' does not name a type
  656 |  i8041ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                       ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:656:65: error: 'device_t' has not been declared
  656 |  i8041ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:656:82: error: 'uint32_t' has not been declared
  656 |  i8041ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                  ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:663:23: error: 'machine_config' does not name a type
  663 |  i8741ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                       ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:663:65: error: 'device_t' has not been declared
  663 |  i8741ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:663:82: error: 'uint32_t' has not been declared
  663 |  i8741ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                  ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:670:21: error: 'machine_config' does not name a type
  670 |  i8042_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:670:63: error: 'device_t' has not been declared
  670 |  i8042_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:670:80: error: 'uint32_t' has not been declared
  670 |  i8042_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:677:21: error: 'machine_config' does not name a type
  677 |  i8742_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                     ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:677:63: error: 'device_t' has not been declared
  677 |  i8742_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                               ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:677:80: error: 'uint32_t' has not been declared
  677 |  i8742_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:684:23: error: 'machine_config' does not name a type
  684 |  i8042ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                       ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:684:65: error: 'device_t' has not been declared
  684 |  i8042ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:684:82: error: 'uint32_t' has not been declared
  684 |  i8042ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                  ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:691:23: error: 'machine_config' does not name a type
  691 |  i8742ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                       ^~~~~~~~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:691:65: error: 'device_t' has not been declared
  691 |  i8742ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                 ^~~~~~~~
../../../../../src/devices/cpu/mcs48/mcs48.h:691:82: error: 'uint32_t' has not been declared
  691 |  i8742ah_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
      |                                                                                  ^~~~~~~~
Compiling src/devices/bus/nes_ctrl/rob.cpp...
../../../../../src/devices/bus/epson_qx/keyboard/keyboard.cpp: In member function 'virtual void bus::epson_qx::keyboard::keyboard_device::device_add_mconfig(machine_config&)':
../../../../../src/devices/bus/epson_qx/keyboard/keyboard.cpp:115:12: error: 'I8049' was not declared in this scope
  115 |  auto &mcu(I8049(config, "mcu", 11_MHz_XTAL));
      |            ^~~~~

```
when compiling on NetBSD 9.99.100/amd64 with gcc 10.4.0.

**Comment by 0-wiz-0** [comment]
Perhaps another issue like
https://github.com/mamedev/mame/issues/9057
where emu.h is missing, and only visible to me since I'm not using pre-compiled headers?

**Comment by happppp** [comment]
I agree with your suspicion.
In that keyboard.cpp, try ordering the includes like this:

```
#include "emu.h"
#include "keyboard.h"
#include "matrix.h"
#include "cpu/mcs48/mcs48.h"

```

**Comment by cuavas** [comment]
Yeah, that’s obviously not going to work with prefix header disabled.  Who even reviewed the associated PR?  (#10384)

Fixed in de835f1b027edc14b866093f4ffaf549e133a329.
